flixel/system/FlxSound.hx

```diff
@@ -713,6 +731,21 @@ class FlxSound extends FlxBasic
 		}
 		return _time = time;
 	}
+	
+	private function set_loopTime(value:Float):Float
+	{
+		return loopTime = FlxMath.bound(value, 0, length);
+	}
+	
+	private function set_loopEndTime(value:Float):Float
+	{
+		return loopEndTime = (value <= loopTime) ? length : value;
```

Comment:
So, you prevent invalid `loopEndTime` values here... However, what these values are depends on `loopTime`. What happens if `loopTime` is set _after_ `loopEndTime`?